.datadog/conf.d/postgres.yaml

```
@@ -0,0 +1,10 @@
+init_config:
+
+instances:
+  - dbm: true
+    host: <WAREHOUSE_MONITOR_HOSTNAME>
+    port: <WAREHOUSE_MONITOR_PORT>
+    username: <WAREHOUSE_MONITOR_USERNAME>
+    password: <WAREHOUSE_MONITOR_PASSWORD>
+    dbname: <WAREHOUSE_MONITOR_DBNAME>
+    ssl: True
```

```suggestion
    service: warehouse
    ssl: True
```
So that metrics are sent to `service:warehouse` instead of `service:metabase` or whatever else the Heroku Datadog buildpack selects by default.

---

https://github.com/DataDog/integrations-core/blob/e660a0f6368642cc4b16e202cff943cb16017a3c/postgres/datadog_checks/postgres/data/conf.yaml.example#L471